spec/models/user_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  before(:each) do
```

Comment:
**Optional** Check the [shoulda](https://github.com/thoughtbot/shoulda) gem to create association tests.
`it { should belong_to(:user) }`